jmbase/jmbase/commands.py

```diff
@@ -157,47 +159,47 @@ class JMRequestMsgSig(JMCommand):
     """Request the client to sign a message-channel
     layer message with the bitcoin key for the nick
     """
-    arguments = [('nick', String()),
-                 ('cmd', String()),
-                 ('msg', String()),
-                 ('msg_to_be_signed', String()),
-                 ('hostid', String())]
+    arguments = [(b'nick', Unicode()),
+                 (b'cmd', Unicode()),
+                 (b'msg', Unicode()),
+                 (b'msg_to_be_signed', Unicode()),
+                 (b'hostid', Unicode())]
 
 class JMRequestMsgSigVerify(JMCommand):
     """Request the client to verify a counterparty's
     message-channel layer message against the provided nick
     """
-    arguments = [('msg', String()),
-                 ('fullmsg', String()),
-                 ('sig', String()),
-                 ('pubkey', String()),
-                 ('nick', String()),
-                 ('hashlen', Integer()),
-                 ('max_encoded', Integer()),
-                 ('hostid', String())]
+    arguments = [(b'msg', Unicode()),
+                 (b'fullmsg', Unicode()),
+                 (b'sig', Unicode()),
+                 (b'pubkey', Unicode()),
+                 (b'nick', Unicode()),
+                 (b'hashlen', Integer()),
+                 (b'max_encoded', Integer()),
+                 (b'hostid', Unicode())]
 
 """ TAKER-specific commands
 """
 
 class JMOffers(JMCommand):
     """Return the entire contents of the
     orderbook to TAKER, as a json-ified dict;
-    note uses BigString because can be very large
+    note uses BigUnicode() because can be very large
```

Comment:
the data type is still called BigString

Reply:
Fixed, I had forgotten to fix that from the find and replace.